### PR TITLE
Update predictionio to latest version

### PR DESCRIPTION
**Pull request by petergarbers**
* Update java version to 1.8

### CODE REVIEW

**Comment by petergarbers** [comment]
https://docs.prediction.io/install/install-linux/ (The instructions there are for mac and linux)

File: Library/Formula/predictionio.rb [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-  sha1 'e5724b637382a06c098bbb3617b35f5bcff1b469'
+  url 'http://download.prediction.io/PredictionIO-0.9.2.tar.gz'
+  sha1 '199d1c9a4b69c3bb009ed7819294183e99ced4e9'
 
   depends_on 'elasticsearch'
   depends_on 'hadoop'
   depends_on 'hbase'
   depends_on 'apache-spark'
-  depends_on :java => "1.7"
+  depends_on :java => "1.8"
```
Comment (xu-cheng) [inline Library/Formula/predictionio.rb]: It cannot work with Java 1.7? If not, it should be ` depends_on :java => "1.7+"`

Comment (petergarbers) [inline Library/Formula/predictionio.rb]: I don't see a reason why it wouldn't work with java 1.6. I am happy making that changve though

**Comment by asparagui** [comment]
Thanks!

**Pull request closed without merging**